plugins/cache.rst

```diff
@@ -33,22 +33,33 @@ control headers from the server as specified in :rfc:`7234`. It needs a
         [$cachePlugin]
     );
 
+.. note::
+
+    Since v1.3.0 does the ``CachePlugin`` have 2 factory methods to easily setup the plugin by caching type.
+
+    .. code-block:: php
+        $cachePlugin = CachePlugin::clientCache($pool, $streamFactory, $options); // Which allows caching responses with the ``private`` or ``no-store`` cache directive
```

Comment:
Can one have code blocks within notes?

Anyhow: make sure to split comments and code so the lines are not too long. 

```php
// comment
$cachePlugin = // ...

// comment
$cachePlugin = // ...
```